pkg/apis/v1beta1/register.go

```diff
@@ -0,0 +1,42 @@
+/*
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1beta1
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+)
+
+const (
+	Group              = "karpenter.sh"
+	ExtensionsGroup    = "extensions." + Group
+	CompatabilityGroup = "compatibility." + Group
+	TestingGroup       = "testing." + Group // Exclusively used for labeling/discovery in testing
```

Comment:
Can we replace with a non-karpenter-specific testing group? Maybe a framework specific group?